### PR TITLE
Update Export-user-data-in-Project-Server-2019.md

### PR DESCRIPTION
**Pull request by NadinMerali**
Adding documentation about an unsupported scenario

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Project/Export-user-data-in-Project-Server-2019.md [hunk 1]
```diff
@@ -637,6 +637,7 @@ After you run the ExportProjectUserContent PowerShell script successfully, you w
 </table>
 
 > **Note:** You can open the .mpp file with Project Professional 2016, Project Professional 2019, or the Project Online Desktop client.
+> Saving the exported .mpp files back to Project Online or Project server is not supported.
```
Comment (JohanFreelancer9) [inline Project/Export-user-data-in-Project-Server-2019.md:640]:
```suggestion
> Saving the exported .mpp files back to Project Online or Project Server is not supported.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @NadinMerali, kindly commit the suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**